src/main/java/org/jitsi/videobridge/AbstractEndpointMessageTransport.java

```diff
@@ -34,6 +34,12 @@
  */
 public abstract class AbstractEndpointMessageTransport
 {
+    /**
+     * The name of the JSON property that indicates the target Octo endpoint id
+     * of a propagated JSON message.
+     */
+    public final String PROP_TARGET_OCTO_ENDPOINT_ID = "targetOctoEndpointId";
```

Comment:
if this is octo-specific, why not put it in `OctoEndpointMessageTransport`?

Reply:
It's actually common between `EndpointMessageTransport` and `OctoEndpointMessageTransport`: When the `EndpointMessageTransport` receives a message from an actual endpoint, it propagates it to all bridges where `OctoEndpointMessageTransport` intercept it.

When the message is propagated, `EndpointMessageTransport` puts the target endpoint id in the message and when the message is intercepted, `OctoEndpointMessageTransport` gets the target endpoint id from the message.

Reply:
got it, i didn't realize this was about propagating from the ingress to egress. thanks.